media/css/pebbles/components/_global-nav.scss

```diff
@@ -0,0 +1,600 @@
+
+            &:after {
+                background-image: url('/media/img/nav/subnav-expand.svg');
+                content: '';
+                height: 12px;
+                position: absolute;
+                right: 0;
+                top: 0;
+                transition: transform .2s ease-in-out;
+                width: 12px;
+            }
+        }
+
+        &.selected {
+            a:link:after,
+            a:visited:after {
+                transform: rotate(45deg);
+            }
+        }
+    }
+
+    .detail-container {
+        border-top: 1px solid #fff;
+        border-bottom: 1px solid #fff;
+        padding-top: 20px;
+    }
+
+    .nav-menu-secondary-links > li {
+        @include font-size(16px);
+        margin: 20px 0;
+
+        a:link,
+        a:visited {
+            display: block;
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Common color styles for both primary and side navigation links & headings
+
+.moz-global-nav,
+.moz-global-nav-drawer {
+
+    .item-firefox {
+        a:hover,
+        a:active,
+        a:focus,
+        a.selected {
+            color: $color-firefox-light-orange;
+        }
+    }
+
+    .item-internet-health {
+        a:hover,
+        a:active,
+        a:focus,
+        a.selected {
+            color: $color-brand-cyan;
+        }
+    }
+
+    .item-technology {
+        a:hover,
+        a:active,
+        a:focus,
+        a.selected {
+            color: $color-brand-lilac;
+        }
+    }
+
+    .item-about-us {
+        a:hover,
+        a:active,
+        a:focus,
+        a.selected {
+            color: $color-brand-lime;
+        }
+    }
+
+    .item-get-involved {
+        a:hover,
+        a:active,
+        a:focus,
+        a.selected {
+            color: $color-brand-coral;
+        }
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Semi-opaque mask shown when side drawer is open.
+
+.moz-global-nav-page-mask {
+    background: #000;
+    bottom: auto;
+    left: 0;
+    opacity: 0;
+    position: absolute;
+    right: auto;
+    top: 0;
+    transition: opacity 0.2s ease-in-out;
+    z-index: 10000;
+}
+
+html.moz-nav-open {
+    .moz-global-nav-page-mask {
+        bottom: 0;
+        opacity: 0.3;
+        right: 0;
+        transition: opacity 0.2s ease-in-out;
+    }
+}
+
+/* -------------------------------------------------------------------------- */
+// Customized Firefox download button for primary navigation.
+
+#global-nav-download-firefox {
+    float: right;
+    margin-right: 25px;
+
+    .download-list {
+        margin: 0;
+
+        &> li {
+            margin: 0;
+        }
+    }
+
+    .download-link:link,
+    .download-link:visited {
+        @include font-size(14px);
+        background: inherit;
+        border-radius: 0;
+        border: 1px solid $color-firefox-light-orange;
+        color: $color-firefox-light-orange;
```

Comment:
Let's try darkening the text a tad for better contrast. Maybe `color: darken($color-firefox-light-orange, 5%);`